packages/react-components/react-combobox/src/components/ComboButton/useComboButtonStyles.ts

```diff
@@ -64,6 +64,7 @@ const useStyles = makeStyles({
     columnGap: tokens.spacingHorizontalXXS,
     display: 'flex',
     flexWrap: 'nowrap',
+    fontFamily: tokens.fontFamilyBase,
```

Comment:
interestingly, the design spec actually doesn't call for any given font-weight 😅. They are all regular weight in practice though.

I can't fully use `typographyStyles` since there isn't a style for regular-weight 400 size text. Given that, I'm kind of inclined to just put the font weight and font family on the base style block, and only alter font-size/line-height for sizes.

Does that make sense to you? That feels less weird to me than having a mixed approach :P

Reply:
It might be worthwhile double checking the font sizes with design. The design spec should probably be referencing the font aliases like "Subtitle 2" instead of just font sizes. In fact, it might be a bug in the design that it's asking for a 400-size font with regular weight, so this might help clear that up.

Reply:
I chatted w/ design, they're checking on whether to add another font alias for this